File: Практика_самостоятельная работа/Python/2.20.py
import math

# Функция для вычисления a
def calculate_a(e, f, g):
    return math.pow(math.sqrt(abs((e - 3) / f)), 3) + g

# Функция для вычисления b
def calculate_b(e, h):
    return math.sin(e) + math.cos(h)**2

# Функция для вычисления c
def calculate_c(e, f, g):
    return 33 * g / (e * f) - 1/3

# Ввод значений e, f, g и h
e = float(input("Введите e: "))
f = float(input("Введите f: "))
g = float(input("Введите g: "))
h = float(input("Введите h: "))

# Вычисление и вывод результатов
a = calculate_a(e, f, g)
b = calculate_b(e, h)
c = calculate_c(e, f, g)
print("a =", a)
print("b =", b)
print("c =", c)

File: Практика_самостоятельная работа/C++/2.20.cpp
#include <iostream>
#include <cmath>

using namespace std;

double calculate_a(double e, double f, double g) {
    return pow(sqrt(abs((e - 3) / f)), 3) + g;
}

double calculate_b(double e, double h) {
    return sin(e) + pow(cos(h), 2);
}

double calculate_c(double e, double f, double g) {
    return 33 * g / (e * f) - 3;
}

int main() {
    double e, f, g, h, a, b, c;

    cout << "Введите e: ";
    cin >> e;
    cout << "Введите f: ";
    cin >> f;
    cout << "Введите g: ";
    cin >> g;
    cout << "Введите h: ";
    cin >> h;

    a = calculate_a(e, f, g);
    b = calculate_b(e, h);
    c = calculate_c(e, f, g);

    cout << "a = " << a << endl;
    cout << "b = " << b << endl;
    cout << "c = " << c << endl;

    return 0;
}
